### PR TITLE
cardano-tracer: logging/monitoring service for node

### PR DESCRIPTION
**Pull request by denisshevchenko**
This PR introduces `cardano-tracer`, the last part of the new tracing infrastructure.

Previously, the node handled all the logging by itself. Moreover, it provided monitoring tools as well: two web-servers, for Prometheus and for EKG monitoring page. `cardano-tracer` is a result of _moving_ all the logging/monitoring-related stuff from the node to a separate service. As a result, the node became smaller, faster, and simpler.

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -15,6 +15,7 @@ packages:
     trace-dispatcher
     trace-resources
     trace-forward
+    cardano-tracer
```
Comment (deepfire) [inline cabal.project:18]: Sorry for the nitpicking, but I imagine people prefer to keep this list sorted.

**Review by deepfire — APPROVED** [review]
Thank you @denisshevchenko

File: cabal.project (hunk 1 above)
Comment (denisshevchenko) [reply cabal.project]: Fixed

**Review by Jimbo4350 — APPROVED** [review]

**Pull request merged**